TeamCode/src/main/java/org/firstinspires/ftc/teamcode/AprilTagDetectionFrameProcessor.java

```diff
@@ -83,31 +88,25 @@ public AprilTagDetectionPipeline(double tagsize, double fx, double fy, double cx
     }
 
     @Override
-    public void finalize()
-    {
+    protected void finalize() {
         // Might be null if createApriltagDetector() threw an exception
-        if(nativeApriltagPtr != 0)
-        {
+        if (nativeApriltagPtr != 0) {
             // Delete the native context we created in the constructor
             AprilTagDetectorJNI.releaseApriltagDetector(nativeApriltagPtr);
             nativeApriltagPtr = 0;
-        }
-        else
-        {
+        } else {
             System.out.println("AprilTagDetectionPipeline.finalize(): nativeApriltagPtr was NULL");
         }
     }
 
     @Override
-    public Mat processFrame(Mat input)
-    {
+    public void processFrame(Mat input, Mat output) {
+
         // Convert to greyscale
-        Imgproc.cvtColor(input, grey, Imgproc.COLOR_RGBA2GRAY);
+        Imgproc.cvtColor(input, grey, Imgproc.COLOR_BGR2GRAY);
```

Comment:
Our `WebCam` class passes input frames as `BGR` format instead of `RGBA`, since that's what the QR code detector needed last year.  We may want to change this in a future PR since it's no longer a requirement.